protocols/interfaces.go

```diff
@@ -7,7 +7,7 @@ import (
 
 // Message is an object to be sent across the wire. It can contain a proposal and signed state hashes, and is addressed to a counterparty.
 type Message struct {
-	To          []byte
+	To          types.Address
```

Comment:
Clarifying question: is the assumption here that a message service should determine **how and where to send a message** based on the _blockchain address_ of the recipient?

Reply:
Maybe - this isn't __necessarily__ intended as a final detail, but I expect:
- everyone that a client will be interested in messaging will own an address. It's probably suitable as an identifier.
- the message service can internally manage a lookup that maps from `types.Address` to transport layer data, possibly something like https://github.com/multiformats/go-multiaddr (used in Lotus / go-fil-markets)